proto/nym/vpn.proto

```diff
@@ -36,6 +36,25 @@ message ExitNode {
   }
 }
 
+message MixConnectedStateDetails {
+  Address nym_address = 1;
+  Address exit_ipr = 2;
+  string ipv4 = 3;
+  string ipv6 = 4;
+}
+
+message WgConnectedStateDetails {
+  string entry_ipv4 = 1;
+  string exit_ipv4 = 2;
+}
+
+message ConnectedStateDetails {
+  oneof connected_state_details_enum {
```

Comment:
Suggestion: `connected_state_details` as `_enum` prefix is redundant (yes oneof fields are converted as Rust enum)